### PR TITLE
Improve the README

### PR DESCRIPTION
**Pull request by ixxie**
A new README for Threlte 6.

Animated banner scoped our because its hard to get it right.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,75 @@
...
-![npm-version](https://img.shields.io/npm/v/@threlte/core?style=flat-square)
-![discord](https://img.shields.io/discord/985983540804091964?label=discord&style=flat-square)
-![website](https://img.shields.io/website?down_color=red&down_message=offline&label=threlte.xyz&style=flat-square&up_message=online&url=https%3A%2F%2Fthrelte.xyz)
+## Create 3D Apps for the Web
 
-Build three.js scenes declaratively with Threlte components that are reactive to the core, readily interactive and can participate in Svelte's ecosystem.
+Threlte is a [Svelte](https://svelte.dev/) library that simplifies creating 3D apps for the web. It provides a **declarative**, **type-safe**, **reactive** and **interactive** API out-of-the-box.
 
-## Official Documentation
+Threlte's core **3D rendering** is powered by [Three.js](https://threejs.org/), but its also provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/).
```
Comment (grischaerbe) [inline README.md:16]: Suggestion:

"Threlte's **3D rendering** is powered by [Three.js](https://threejs.org/), it provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/)."

File: README.md [hunk 2]
```diff
@@ -1,38 +1,75 @@
...
+
+## Features
+
+ - [Clean API](https://threlte.xyz/docs/reference/core/t) with a single component transparently exposing *all* Three.js objects through declarative API.
+
+ - [Hooks](https://threlte.xyz/docs/learn/basics/hooks) simplifying access to the underlying Three.js contexts such as animation frames and the rendering engine.
+
+ - [Plugins](https://threlte.xyz/docs/learn/advanced/plugins) making it easy to extend Threlte with custom code and logic.
+
+ - [Events](https://threlte.xyz/docs/learn/basics/events) lets you access state transformations in your scenes.
```
Comment (grischaerbe) [inline README.md:48]: This sounds a bit odd, maybe "Events enabling robust and Svelte-native access to state transformations occurring within your scenes." or along these lines?

File: README.md [hunk 3]
```diff
@@ -1,38 +1,75 @@
...
+
+ - [Interactivity](https://threlte.xyz/docs/reference/extras/interactivity) makes it easy to drive state with user actions on 3D objects.
```
Comment (grischaerbe) [inline README.md:50]: "… makes it easy to react to user input …"

**Review by grischaerbe — CHANGES_REQUESTED** [review]
Thanks again, looks and reads so much better than before ❤️ 

Found some smaller things here and there.
Also: Can you replace all links to "https://threlte.xyz" with "https://next.threlte.xyz" so it points to the next page? When merging to main (aka releasing T6) we can get rid of all "next" occurrences in one go 👍

**Comment by ixxie** [comment]
@grischaerbe I addressed all the issues, but sometimes in somewhat different ways you suggested. I also streamline some other text a bit to shorten it. Can you give it a read through again to see if you are good with all of it?

**Comment by grischaerbe** [comment]
LGTM 🚀 Thank you so much!

**Pull request merged**